fabric-client-gametest-api-v1/src/client/java/net/fabricmc/fabric/api/client/gametest/v1/TestServerContext.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.gametest.v1;
+
+import org.apache.commons.lang3.function.FailableConsumer;
+import org.apache.commons.lang3.function.FailableFunction;
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.server.MinecraftServer;
+
+/**
+ * Context for a client gametest containing various helpful functions while a server (integrated or dedicated) is
+ * running.
+ *
+ * <p>Functions in this class can only be called on the client gametest thread.
+ */
+@ApiStatus.NonExtendable
+public interface TestServerContext {
+	/**
+	 * Runs a command on the server.
+	 *
+	 * @param command The command to run
+	 */
+	void runCommand(String command);
```

Comment:
A thing for the future, but getting the result of the command might be handy, not quite how it would work but I could see it being useful.

Reply:
This function was just copied over from `TestDedicatedServer`. But yes, could be handy. Annoyingly the current function it's implemented with doesn't return the result of the command but it should hopefully be a case of finding a different function to use.